README.md

```diff
@@ -18,24 +18,27 @@
    Cmd + P and select 'Dev containers: rebuild containers without cache`.
 2. Then you can start querying the API at `http://localhost:8889`
 
-### Usage
+---
 
-Start the webserver project:
+## Usage
 
+### Start the webserver project:
+This will watch the project directory and restart as necessary.
 ```
 yarn chat
 ```
 
-Make sure that `sessions` directory exists under `workspace`, otherwise do
-
+### Start the UI component:
 ```
-mkdir sessions
+yarn chat-styles
 ```
 
-This will watch the project directory and restart as necessary.
-
-Start the UI component:
+### Format the deno TS sourcecode:
+```
+yarn lint
+```
```

Comment:
🙌

Reply:
there are some cases of errors with `deno.lock` that can be solved by _clearing the cache_
... I though exposing it in the _root-level_ scripts might help is trouble arises.

About the PR template I'm all willing to adopt your proposition, but was feeling lazy tonight and thought the _Why_ + _What was done_ structure could convey that small PR intent.